fence/blueprints/link.py

```diff
@@ -442,7 +456,13 @@ def _force_update_user_google_account(
                 "was attempted and failed."
             )
 
+    # timestamp at which the link will expire
     expiration = get_default_google_account_expiration()
+    if "google_link_expires_in" in flask.session:
+        requested_expiration = int(time.time()) + flask.session.pop(
+            "google_link_expires_in", None
```

Comment:
```suggestion
            "google_link_expires_in", expiration-int(time.time())
```

I don't think Python likes adding `None` to an int

Reply:
oh, but you already check if it's in the session.. so I think you can just pop it safely without a default

Reply:
```suggestion
            "google_link_expires_in"
```

Reply:
pop requires a default -> will fix that separately

Reply:
`_clear_google_link_info_from_session` is called before the param is used here: https://github.com/uc-cdis/fence/blob/master/fence/blueprints/link.py#L273-L284
I can either leave it as it is (since it's popped as soon as it's used it should be okay?) or add it as parameter to `_force_update_user_google_account`